controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -145,7 +145,12 @@ def run_match(supervisor: Supervisor) -> None:
         robot.getField('customData').setSFString('start')
 
     # ... then un-pause the simulation, so they all start together
-    supervisor.simulationSetMode(Supervisor.SIMULATION_MODE_RUN)
+    if supervisor.getDevice("2021a-compatibility") is None:
+        # we are running version 2020b so the old command is used
+        supervisor.simulationSetMode(Supervisor.SIMULATION_MODE_RUN)
+    else:
+        # webots-2021a removed the RUN mode and now uses FAST
+        supervisor.simulationSetMode(Supervisor.SIMULATION_MODE_FAST)
```

Comment:
Do we actually want to prevent curious teams from being able to run an actual match because they are using a newer version?
Could we just output a clear warning that their environment doesn't match the competition setup, such as:
```suggestion
        print("All offical competition matches will be run on version 2020b-rev1 unless otherwise stated", file=sys.stderr)
        print("Since this simulator is running a different version some behaviour may not match that of the offical competition matches", file=sys.stderr)
        supervisor.simulationSetMode(Supervisor.SIMULATION_MODE_FAST)
```

Reply:
Minor tweaks and typo fixes
>All official competition matches will be run on Webots version 2020b-rev1 unless otherwise stated
>
>Since this simulator is running a different version it is possible that some behaviour may not match that of the official competition matches

Reply:
That does read better so I'm happy to use that

Reply:
Personally i'd rather we mention the competition version in the docs rather than the simulator logs. It makes things easier to change in future, and keeps the logs clean.

Also, are we sure the _competition_ will run 2020b-rev1? I thought we had decided the first league session would, and we'd keep an eye from there?

Reply:
> Personally i'd rather we mention the competition version in the docs rather than the simulator logs. It makes things easier to change in future, and keeps the logs clean.

We could reduce the warning to just:
```
This simulator is running a different version than is being used for the next official competition matches, please check the docs for which version will be used
As such it is possible that some behaviour may not match that of the official competition matches
``` 

> Also, are we sure the _competition_ will run 2020b-rev1? I thought we had decided the first league session would, and we'd keep an eye from there?

Agreed we could do with being more vague about this to give ourselves options

Reply:
I like the reduced warning. Below is how I would personally tweak it but feel free to take or leave any changes as this is getting quite nit picky.
```
This simulator is running a different version of Webots to the one that will be used for the next official competition matches (You can check the docs to see which version will be used)
As such it is possible that some behaviour may not match that of the official competition matches
```

Reply:
see 850b6e3f9a031f0d7187c490e3837875c9d8b00d